lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -388,7 +391,7 @@ instance Arbitrary Tx where
         outs <- fmap (L.take 5 . getNonEmpty) arbitrary
         wdrls <- fmap (Map.fromList . L.take 5) arbitrary
         fees <- arbitrary
-        mkTx fees ins cins outs wdrls <$> arbitrary
+        mkTx fees ins cins outs wdrls <$> arbitrary <*> arbitrary
```

Comment:
I tend to think all these calls to `arbitrary` make it harder than necessary to figure out exactly which generator is being used.

Suggestion:
```suggestion
        mkTx fees ins cins outs wdrls <$> arbitrary <*> genTxScriptValidity
```

Reply:
1105933ed6bcdd7d26ca9fb5d4ea4e7e7b143459